platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/views/LearnerTreeView.java

```diff
@@ -35,15 +40,17 @@
  */
 public interface LearnerTreeView<T> extends LearnerExpectedLessonQueue<T>, LearnerInitializer<T>, TreeView<T> {
 
-    /**
-     * Start any required background threads. Threads should be created on the provided work group.
-     *
-     * @param threadManager
-     * 		responsible for creating new threads
-     * @param workGroup
-     * 		a work group responsible for managing threads
-     */
-    default void startThreads(final ThreadManager threadManager, final StandardWorkGroup workGroup) {}
```

Comment:
Might want to retain the javadoc and update it accordingly.

Reply:
The method is gone. Why to retain the JavaDoc?

Reply:
It's not gone. It's replaced with `void startLearnerThreads(...)` right at this place, and this new method doesn't have any javadoc. So I'm suggesting to write one.